src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -80,6 +84,16 @@ const TableHeroImage = ({
         }
       </div>
     }
+    { !!showPvgnaGuide && pvgnaGuideInfo && heroName &&
```

Comment:
I find Boolean() more readable if I need to make sure something's a certain type